docs/guides/cdk/using-amplify-with-cdk.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Using Amplify with CDK
+description: How to use existing Amplify resources in a CDK project
+---
+
+When using Amplify, you may run into a scenario when you need a feature that the Amplify CLI does not yet support. As an escape hatch, you can either configure [custom resources](~/cli/usage/customcf.md) with Amplify or manage resources outside of Amplify using another cloudformation provider.
+
+In this guide you'll learn how to extend an Amplify project with CDK by importing existing Amplify resources into a CDK project. You'll then learn how to programmatically create a CDK exports file similar to the Amplify exports file for consumption by your client-side project.
+
+### Getting started
+
+This guide will walk you through adding authentication using the Amplify CLI and then building a GraphQL API that references and uses this authentication service as an authentication and authorization provider.
+
+## Creating the Amplify app
+
+The first step will be to initialize an Amplify app and create a new authentication service.
+
+<amplify-callout>
+
+If you already have an authentication service created using the Amplify CLI, you can jump to [Creating the CDK project](#creating-the-cdk-project)
+
+</amplify-callout>
+
+### Initializing the Amplify project:
+
+```sh
+amplify init
+```
+
+### Adding authentication
+
+```sh
+amplify add auth
+
+? Do you want to use default authentication and security configuration? Default configuration 
+? How do you want users to be able to sign in when using your Cognito User Pool? Username
+? Do you want to configure advanced settings? No, I am done. 
+```
+
+### Deploying the Amplify back end
+
+```sh
+amplify push --y
+```
```

Comment:
instead of using a bunch of headers for this relatively common task i suggest collapsing these instructions

```suggestion
```sh
amplify init # Initializing the Amplify project - choose all defaults
amplify add auth # Adding authentication

? Do you want to use default authentication and security configuration? Default configuration 
? How do you want users to be able to sign in when using your Cognito User Pool? Username
? Do you want to configure advanced settings? No, I am done. 

amplify push --y # Deploying the Amplify back end
```
```

Reply:
Thanks Shawn, good input.